openedx/features/redhouse_panel/api/v0/views.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.sites.models import Site
+from django.shortcuts import get_object_or_404
+from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from rest_framework_oauth.authentication import OAuth2Authentication
+
+from openedx.core.djangoapps.site_configuration.models import SiteConfiguration
+from openedx.core.lib.api.permissions import IsStaffOrOwner
+
+from .serializers import SiteSerializer
```

Comment:
Please avoid using relative imports

Reply:
@danialmalik this is an explicit relative import which will make this app more portable. let me know if I'm missing someting

Reply:
I get it but we generally don't use relative imports to keep things consistent.

Reply:
@sheralim012 We mostly avoid using relative imports in edX and this app is totally dependant on the edX so there is no need to use relative imports. Please use absolute imports.

Reply:
@danialmalik understood and done